dotcom-rendering/stylelint.config.mjs

```diff
@@ -0,0 +1,35 @@
+/** @type {import('stylelint').Config} */
+export default {
+	extends: ['stylelint-config-recommended'],
+	customSyntax: 'postcss-styled-syntax',
```

Comment:
[As per the docs](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#removed-processors-configuration-property)